<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovToExcel</class>
 <widget class="QDialog" name="MovToExcel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>项目Excel生成工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>素材路径: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="path_line"/>
         </item>
         <item>
          <widget class="QToolButton" name="path_btn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>生成位置: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="excel_path_line"/>
         </item>
         <item>
          <widget class="QToolButton" name="excel_path_btn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="to_excel_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>57</height>
        </size>
       </property>
       <property name="text">
        <string>生成Excel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
